docs/source/release/v6.1.0/muon.rst

```diff
@@ -5,21 +5,13 @@ MuSR Changes
 .. contents:: Table of Contents
    :local:
 
-.. warning:: **Developers:** Sort changes under appropriate heading
-    putting new features at the top of the section, followed by
-    improvements, followed by bug fixes.
-
-
 Muon Analysis 2 and Frequency Domain Analysis
 ---------------------------------------------
 
 New Features
 ############
 - Added a new "periods" button to the grouping tab which displays a table of information on periods (currently only supporting nexus V1 files).
 
-Improvements
```

Comment:
we should have the fitting improvements in this section

Reply:
I haven't deleted any lines related to fitting in this file. There are two fitting related things in the `framework.rst` were these intended for muons?
```
- New algorithm :ref:`GeneratePythonFitScript <algm-GeneratePythonFitScript>` allows the creation of a python script for sequential fitting.
- New algorithm :ref:`ProfileChiSquared1D <algm-ProfileChiSquared1D>` to profile chi squared after a fit. This can be used to find better estimates of parameter errors.
```

Reply:
No, so I think it just got missed from the release notes. It is just that "the moun and frequency domain analysis GUI's fitting interface has been improved"

Reply:
Perhaps we could add some examples e.g. phase quad stuff (just looking through closed PRs to find inspiration!)?

Reply:
to the improvements section?

Reply:
To clarify - can you think of any specific improvements that have been made to the muon and frequency domain analysis fitting interface. It's OK if not, I will go with what you suggested above.

Reply:
I will have a good think after the group meeting